src/cloud-api-adaptor/pkg/cdh/config.go

```diff
@@ -21,8 +21,9 @@ type Config struct {
 }
 
 type KBCConfig struct {
-	Name string `toml:"name"`
-	URL  string `toml:"url"`
+	Name    string `toml:"name"`
+	URL     string `toml:"url"`
+	KBSCERT string `toml:"kbs_cert"`
```

Comment:
```suggestion
	KBSCert string `toml:"kbs_cert"`
```

Reply:
I think we want to add `omitempty` to the cert field annotatuion so that an empty string will not be serialized.